websites/X/X.com/metadata.json

```diff
@@ -0,0 +1,41 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"name": "Bas950",
+		"id": "241278257335500811"
+	},
+	"service": "X.com",
+	"altnames": ["Twitter"],
+	"description": {
+		"en": "From breaking news and entertainment to sports and politics, from big events to everyday interests. If it's happening anywhere, it's happening on X.",
+		"nl": "Van het laatste nieuws en entertainment tot sport en politiek, van grote evenementen tot dagelijkse interesses. Als het ergens gebeurt, gebeurt het op X.",
+		"zh_CN": "X是一家美国社交网络及微博客服务的网站，是全球互联网上访问量最大的十个网站之一。",
+		"zh_TW": "X是一個社群網路與微部落格服務。",
+		"vi_VN": "Từ các tin nóng, đến giải trí, đến các đề tài thể thao và chính trị, từ các sự kiện lớn tới các thú vui hằng ngày. Nếu nó đang xảy ra ở bất cứ đâu, nó đang xảy ra trên X."
+	},
+	"url": ["twitter.com", "x.com"],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/M2cZMdq.png",
+	"thumbnail": "https://i.imgur.com/ogWoKqI.png",
+	"color": "#000000",
+	"category": "socials",
+	"tags": ["social", "media", "twitter"],
+	"settings": [
+		{
+			"id": "lang",
+			"multiLanguage": true
```

Comment:
```suggestion
			"multiLanguage": "x"
```

Reply:
Think this is the best way of solving it, doesn't make sense to rename all strings again. Especially since the service name includes a dot, will make it confusing. @theusaf @DarkVillager @rusprice agreed?

Reply:
That makes sense to me.